Pod/Classes/DictionaryConvenience.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DictionaryConvenience.swift
+//  Pods
+//
+//  Created by Matt Thomas on 5/20/16.
+//
+//
+
+import Foundation
+
+public func += <Key, Value> (inout left: [Key: Value], right: [Key: Value]) {
```

Comment:
That's an interesting idea. I also want to add something to this PR that I use in Restock (`+` operator) so hold off on this PR at the moment.